chunk.go

```diff
@@ -25,38 +25,34 @@ func (ch *chunk) add(record []byte) {
 
 // dump the chunk into w, and clears the chunk and makes it ready for
 // the next add invocation.
-func (ch *chunk) dump(w io.Writer, compressorIndex int) error {
+func (ch *chunk) dump(w io.Writer, compressorID int) error {
 	// NOTE: don't check ch.numBytes instead, because empty
 	// records are allowed.
 	if len(ch.records) == 0 {
 		return nil
 	}
 
 	// Write raw records and their lengths into data buffer.
-	var data bytes.Buffer
```

Comment:
This `var data bytes.Buffer` is not necessary. This PR removes it and keeps only `compressed`.